README.md

```diff
@@ -1,9 +1,166 @@
+| request.header.content_length | If this field has the value ```auto``` then its value is calculated from the generated request content.  If the value is a number, then that value is used in the field instead. |
+| request.header.auth_length | If this field has the value ```auto``` then its value is calculated from the generated auth content.  If the value is a number, then that value is used in the field instead. |
+| request.body_description | A free form description of the contents of the request content.  Used for test writers (and generator writers) to understand how to construct the request object.  This field is not used by the test generator. |
+| request.auth.type | This can either be ```none```, which will cause the auth section of the message to be empty (equivalent of the No Authentication type of authentication); or it can be ```direct```, which will cause the auth section of the message to contain authentication data corresponding to the format for Direct Authentication.  If ```direct``` is set, then the request.auth.app_name field must be set.  Note that this field does not cause the header auth_type field to be set. |
+| request.auth.app_name | Used to populate the auth section of the message when ```direct``` authentication is selected |
+| result | Settings for configuration the result data for the test data file |
+| result.header | Field values for the result header.  Meanings of these files and valid values can be found in the [parsec book](https://parallaxsecond.github.io/parsec-book/parsec_client/wire_protocol.html). |
+| result.header.content_length | If this field has the value ```auto``` then its value is calculated from the generated result content.  If the value is a number, then that value is used in the field instead |
+| result.header.auth_length | If this field has the value ```auto``` then its value is calculated from the generated auth content.  If the value is a number, then that value is used in the field instead. |
+| result.body_description | A free form description of the contents of the result content.  Used for test writers (and generator writers) to understand how to construct the request object.  This field is not used by the test generator. |
+| result.auth.type | This can either be ```none```, which will cause the auth section of the message to be empty (equivalent of the No Authentication type of authentication); or it can be ```direct```, which will cause the auth section of the message to contain authentication data corresponding to the format for Direct Authentication.  If ```direct``` is set, then the result.auth.app_name field must be set.  Note that this field does not cause the header auth_type field to be set.  **NOTE:**  A Parsec client would not send authentication data in a result message, but this spec format allows test authors to create the message as they wish to excersice the parsec client code. |
```

Comment:
```suggestion
| result.auth.type | This can either be ```none```, which will cause the auth section of the message to be empty (equivalent of the No Authentication type of authentication); or it can be ```direct```, which will cause the auth section of the message to contain authentication data corresponding to the format for Direct Authentication.  If ```direct``` is set, then the result.auth.app_name field must be set.  Note that this field does not cause the header auth_type field to be set.  **NOTE:**  A Parsec service would not send authentication data in a result message, but this spec format allows test authors to create the message as they wish to excersice the parsec client code. |
```